data/themes/darktable.css

```diff
@@ -759,15 +759,33 @@ dialog
 
 dialog .dialog-vbox
 {
-  margin: 0.56em;
+  margin: 0.56em 0.42em;
+}
+
+/* set top margin in bottom button bar to be the same as bottom margin applied above for main dialog window */
+dialog .dialog-action-box
+{
+  margin-top: 0.42em;
+}
```

Comment:
I think this is better as 0.56em to mirror the top/bottom margins on the vbox, and then add `#modulegroups-reset {  margin-right: 0.56em; }` as well, the reset button looks a little better. Not a perfect solution though I'll grant you.

Reply:
top/bottom margins are with 0.42em too. You can see that I have change that, just because 0.56em was too much margin. This line is same size as top/bottom margins on the vbox, like my comment explain.

About adding margin-right, I'm against. The issue is not to add that (this would be just a workaround) but be sure having a consistent way to display bottom buttons (with or without action boxes, no matter) and so avoid adding such tricks.

Reply:
Oups, just see that I made a mistake in vbox above by revert top/bottom vs left/right margins... Last update fix that (and so reset button in module manager is also better with that fix.